### DISCUSSION TITLE
PostgresSQL version

### DISCUSSION THREAD

**Discussion by mateusz5ip** [requarks/wiki#7421] [Help / Questions]
I just install the WikiJS on Kubernetes using Helm. I've have a question why the postgresql component is that old.
Fresh installation open a postgresql in version 11.22.0-debian-11-r4 which was released a year ago.
I manually change the image postgresql vestion to 17.0.0-debian-12-r10. Works fine.
Suggest to update to avoid vulnerabilities.

**Comment by BPowell76** [comment]
I don't use Wiki.js in a containerized manner. In doing some digging the charts project used for setting up the Kubernetes clusters has been [archived/deprecated since 2020](https://github.com/helm/charts/tree/master?tab=readme-ov-file#%EF%B8%8F-deprecation-and-archive-notice). I think the preferred method for containerization now is Docker for just working out of the box with Wiki.js.
